parser/raylib_api.lua

```diff
@@ -2636,6 +2636,14 @@ return {
         {name = "height", type = "int"}
       }
     },
+    {
+      name = "SetWindowOpacity",
+      description = "Set window opacity [0.0f..1.0f] (only PLATFORM_DESKTOP)",
+      returnType = "void",
+      params = {
+        {name = "opacity", type = "float"}
```

Comment:
Maybe we could exchange order of those lines to follow C convention of (`type` `name`).